packages/react/package.json

```diff
@@ -17,7 +17,7 @@
     "test:snap:update": "jest --updateSnapshot"
   },
   "dependencies": {
-    "@clr/core": "^3.1.0",
+    "@clr/core": "^4.0.0-rc.0",
```

Comment:
Why is the `4.0.0-rc.1` version of `@clr/react` relying on the `4.0.0-rc.0` version of `@clr/core`?